Modules/SubModule_ASL/xASL_wrp_ResampleASL.m

```diff
@@ -230,8 +230,14 @@ function xASL_wrp_ResampleASL(x)
         
         % Decoding of TimeEncoded data ===================
     elseif x.modules.asl.bTimeEncoded 
-            xASL_quant_HadamardDecoding(x.P.Path_rdespiked_ASL4D, x.modules.asl, x.TimeEncodedEchoTimes); % Nifti is saved inside the function
+            ASL_im = xASL_quant_HadamardDecoding(x.P.Path_rdespiked_ASL4D, x.modules.asl, x.TimeEncodedEchoTimes); % Nifti is saved inside the function
         % =============================================
+        
+        % Save PWI4D
+        fprintf('%s', [PathPWI4D{iSpace} ', ']);
+        xASL_io_SaveNifti(PathASL{iSpace}, PathPWI4D{iSpace}, ASL_im, 32, false);
+        
+        

```

Comment:
For the `fprintf` you probably need to add a `'\n'` somewhere, but I'd have to run this myself to see how it looks. Super minor thing however.

Reply:
done!